packages/xo-web/src/xo-app/logs/backup-ng/index.js

```diff
@@ -49,10 +51,38 @@ export const LogStatus = ({ log, tooltip = _('logDisplayDetails') }) => {
   )
 }
 
+const GoToJob = decorate([
+  withRouter,
+  provideState({
+    effects: {
+      goTo() {
+        const { jobId, location, router, scrollIntoJobs } = this.props
+        router.replace({
+          ...location,
+          query: { ...location.query, s: `id:${jobId}` },
+        })
+        scrollIntoJobs()
+      },
+    },
+  }),
+  injectState,
+  ({ effects, children }) => (
+    <Tooltip content={_('goToThisJob')}>
+      <p onClick={effects.goTo} style={{ cursor: 'pointer' }}>
+        {children}
+      </p>
+    </Tooltip>
+  ),
+])
+
 const COLUMNS = [
   {
     name: _('jobId'),
-    itemRenderer: log => log.jobId.slice(4, 8),
+    itemRenderer: (log, { scrollIntoJobs }) => (
+      <GoToJob jobId={log.jobId} scrollIntoJobs={scrollIntoJobs}>
```

Comment:
`component: GoToJob,`  and you can use `item` and `userData` props in `GoToJob`.

Reply:
With the `itemRendered` syntax we explicit what props needed for `GoToJob`. IMHO, it's better than the component syntax.

Reply:
ok and you can add `propTypes` for `GoToJob`.
What do you think about using `render-xo-item\BackupJob`?

Reply:
`render-xo-item\BackupJob` don't respond to our need because:
1. the column is named `id` so we should only show the id with the ability to copy it
2. the render-xo-item don't give a way to scroll into the backup view